business_objects/monitor.py

```diff
@@ -7,8 +7,15 @@
 from submodules.model.cognition_objects import (
     macro as macro_db_bo,
     markdown_file as markdown_file_db_bo,
+    file_extraction as file_extraction_db_bo,
+    file_transformation as file_transformation_db_bo,
 )
 
+FILE_CACHING_IN_PROGRESS_STATES = [
```

Comment:
maybe in the future we should use sets instread of lists for IN checks. At least from a theoretically standpoint a set would be faster for `in` checks. In practice with two entries it shouldn't even be noticable => nothing to do atm